snuba/datasets/configuration/issues/storages/search_issues.yaml

```diff
@@ -0,0 +1,67 @@
+version: v1
+kind: writable_storage
+name: search_issues
+
+storage:
+  key: search_issues
+  set_key: search_issues
+
+schema:
+  columns:
+    [
+      { name: organization_id, type: UInt, args: { size: 64 } },
```

Comment:
What is the rationale for adding the organization id to the table when we already have project id? 
Today all errors and transactions queries are based on project id alone and things are ok. 

Assuming org id will be in the order by key, if you add the organization id, you will always have to add a condition based on the organization to all your query, which in turn, will imply clickhouse will always load that column when processing your query, which has a performance impact on short and fast queries. 
This impact is generally useless as you already have the project id that serves the same purpose. 

Are there queries where you want to scan all project in an org without filtering them? Those will not be very efficient if we rely on organization id alone. This is because project id will be in the order by key, but it will not be in the query, thus no further field from the order by key will be used to narrow down the scope (including timestamp).

So I would suggest having only project id unless you plan to add organization without adding it to the order by key.

Reply:
Thanks for the context. I think dropping `organization_id` from the order by makes sense. I don't think we have any use-cases to scan all projects within an org. Would we be able to efficiently support something like that later on if we use `project_id` as the first order by column?

Reply:
It looks like eventstream send the `organization__id` as part of the payload. I don't think we have any plans to make queries aggregated by the `organization_id` but filling in the column value doesn't look like a problem. I think we can keep it for now and drop it later instead of removing it now and potentially adding it back in the future.